l10n_it_website_sale_corrispettivi/tests/test_website_sale_no_corrispettivi.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests import HttpCase
+
+
+class TestFrontend(HttpCase):
+
+    def setUp(self):
+        super().setUp()
+        self.admin_partner = self.env.ref('base.partner_root')
+
+    def test_admin_no_corrispettivi(self):
+        """
+        If a partner having the flag 'corrispettivi' set to False
+        creates an order in frontend, then
+        the created order's flag corrispettivi is False"""
+        self.assertFalse(self.admin_partner.use_corrispettivi)
+        existing_orders = self.env['sale.order'].search([])
+        # In frontend, create an order with flag corrispettivi
+        self.phantom_js(
```

Comment:
Curiosity: wouldn't be possible call directly https://github.com/odoo/odoo/blob/11.0/addons/website_sale/tests/test_sale_process.py#L9 ?

Reply:
@tafaRU I think that would be possible but that might be not a good idea, another test (in this case https://github.com/odoo/odoo/blob/11.0/addons/website_sale/tests/test_sale_process.py#L9) might have its own setup that does not get called if I simply write `test_sale_process()`

Reply:
Ok, thanks for the explanation.